docs/api-docs/partner/subscriptions/overview.md

```diff
@@ -0,0 +1,14 @@
+# Introduction to Subscriptions
+
+Merchants can add "Subscribe and save" capabilities to their stores with third-party apps. To build a subscription management app, check out our [open source application framework](https://github.com/bigcommerce/subscription-foundation) and consult our [Subscription Foundation Guide]() to contemplate or configure your own implementation.
+
+Installing a subscription app on its own is only one half of the equation; the merchant must also configure their store to support subscription sales.
```

Comment:
make sure you use the right voice and audience. Some portions of this section say "you" and then "the merchant". Are those 2 different personas? Or is "you" the merchant?